### DISCUSSION TITLE
Display GeoTiff from GeoServer in Timeseries

### DISCUSSION THREAD

**Discussion by S-mardii** [TerriaJS/terriajs#6850] [Q&A]
Hello, my organization have multiple GeoTiff layers hosted on GeoServer and we would like to display those layers as Timeseries on Terria Map.

I have tried to look for the documentation related to this, but I couldn't fund information related to this need.

These are the layers that we plan to work with: https://geoserver.opendevelopmentmekong.net/geoserver/web/wicket/bookmarkable/org.geoserver.web.demo.MapPreviewPage?1&filter=false

Any advice or suggest is highly appreciated.

Best regards,

**Comment by sixlighthouses** [comment]
Hi
thanks for taking the time to try out Terria's time series functionality.  I had a quick look at the geoserver link you posted, you have a lot of data in there!.  Could you please let me know specifically which layer/s you are the geotiffs you wish to use in time series

kind regards

Brendon

**Comment by S-mardii** [comment]
@sixlighthouses Thank you so much for your prompt response, and sorry that I didn't recheck the attached link.

You can use this keyword in the Search Box of Geoserver `forest20`

The following is the screenshot of the layers that I would like to make it as timeseries:
![image](https://github.com/TerriaJS/terriajs/assets/18528756/5631aab8-6651-4d69-86bc-141e8603bfb6)

Thank you

**Comment by sixlighthouses** [comment]
Hi @S-mardii 

If i understand correctly you would like to display the data listed about as a single time series dataset?  If that is correct you can set your data up with the following 

https://docs.geoserver.org/2.23.x/en/user/tutorials/imagemosaic_timeseries/imagemosaic_timeseries.html

please let us know how you get on

thanks

Brendon

**Reply by S-mardii** [reply]
Thank you @sixlighthouses. This is very helpful. My team and I will surely check this out.

By the way, this should mean that when I have the time series published in GeoServer, the time series feature in Terria Map GUI will become available?

![image](https://github.com/TerriaJS/terriajs/assets/18528756/cb755193-6574-4d00-9562-51b208171f0d)

Thank you
